Sprint-2/errors/0.js

```diff
@@ -3,7 +3,8 @@
 // call the function capitalise with a string input
 // interpret the error message and figure out why an error is occurring
 
+
 function capitalise(str) {
-  let str = `${str[0].toUpperCase()}${str.slice(1)}`;
+  str = `${str[0].toUpperCase()}${str.slice(1)}`; // No `let`, just reassign
```

Comment:
Alternatively, you can declare a new variable or just return the value of the expression (without storing the value in a variable first).